### ISSUE TITLE
how to Fine-tuning inception v2 model?

### ISSUE THREAD

**Issue by civilman628** [tensorflow/models#862]
I can see the link below have a command about how to fine tune V3 model, but I need to know how to fine tune V2.

https://github.com/tensorflow/models/tree/master/slim

$ python train_image_classifier.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=flowers \
    --dataset_split_name=train \
    --model_name=inception_v3 \
    --checkpoint_path=${CHECKPOINT_PATH} \
    **--checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits \
    --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits**

How can I use V2 check point file instead? I mean how to update the parameters in bold section, just update InceptionV3 to InceptionV2 and update --model_name= inception_v2   ?  all other parts do not need to change?

**Comment by michaelisard** [comment]
This type of usage question is better asked on [Stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow).
